app/Blocklyduino.php

```diff
@@ -0,0 +1,43 @@
+<?php
+// Blocklyduino.php
+
+namespace app;
+
+use Silex\Application;
+
+class Blocklyduino extends Application {
+
+    /**
+     * Add some traits that make our Application special
+     */
+    use Application\TwigTrait;
+    use Application\UrlGeneratorTrait;
+
+    /**
+     * Instantiate a new Blocklyduino Application.
+     * Objects and parameters can be passed as argument to the constructor.
+     * @param array $values The parameters or objects.
+     */
+    function __construct(array $values = array()) {
+        parent::__construct();
+
+        // Set to false if you don't want debug messages
+        $this['debug'] = true;
+
+        // The URL to use to connect to the Codebender Builder service
+        $this['builder_url'] = "http://builder.codebender.cc:8080/compile?api_key=blocklyduino";
```

Comment:
This should definitely be moved out of here. We need to use some parameters file.
In order to do that, use [this Silex component](https://github.com/deralex/YamlConfigServiceProvider). You will find everything you need in the YamlConfigServiceProvider documentation.